cockatrice/src/dlg_update.cpp

```diff
@@ -47,11 +45,9 @@ DlgUpdate::DlgUpdate(QWidget *parent) : QDialog(parent) {
     //Check for SSL (this probably isn't necessary)
     if (!QSslSocket::supportsSsl()) {
         enableUpdateButton(false);
-        QMessageBox::critical(
-                this,
-                tr("Error"),
-                tr("Cockatrice was not built with SSL support, so cannot download updates! "
-                           "Please visit the download page and update manually."));
+        QMessageBox::critical(this, tr("Error"),
+            tr("Cockatrice was not built with SSL support, so cannot download updates! "
```

Comment:
"Cockatrice was not build with SSL support, so you cannot download updates automatically! Please visit the download page to update manually."